packages/passport/sdk/src/guardian/guardian.ts

```diff
@@ -11,11 +12,11 @@ import { PassportConfiguration } from '../config';
 export type GuardianClientParams = {
   confirmationScreen: ConfirmationScreen;
   config: PassportConfiguration;
+  authManager: AuthManager;
 };
 
 export type GuardianEvaluateImxTransactionParams = {
   payloadHash: string;
-  user: UserImx;
 };
 
 type GuardianEVMValidationParams = {
```

Comment:
We can, the reason I passed through the user is because the only call to `evaluateEvmTransaction` is on the very next line after we retrieve the user in `validateEvmTransaction`, which is something we need to do in order to provide the address for `confirmationScreen.requestConfirmation`, so it seemed redundant. Happy to change it for consistency if you'd prefer though.

Reply:
They were still being passed in, but I'll remove it as part of the other requested change